### PR TITLE
Feat/demography

### PR DESCRIPTION
**Pull request by giovannic**
This PR implements a custom demography for a constant population.

It would be cool to see if the Demography.Rmd vignette makes sense, or if it could be explained more clearly.

### CODE REVIEW

File: vignettes/Demography.Rmd [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+suppressPackageStartupMessages(library(ggplot2))
```
Comment (htopazian) [inline vignettes/Demography.Rmd:18]: ggplot??!?!?!?! yes!

File: vignettes/Demography.Rmd [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+# Parameterisation
+
+We are going to create a scenario where we track severe prevalence for 5 years.
+
+```{r}
+year <- 365
+month <- 30
+sim_length <- 5 * year
+human_population <- 1000
```
Comment (htopazian) [inline vignettes/Demography.Rmd:30]: this is a great example - super clear.

File: R/parameters.R [hunk 3]
```diff
@@ -405,7 +407,9 @@ get_parameters <- function(overrides = list()) {
     severe_incidence_rendering_min_ages = numeric(0),
     severe_incidence_rendering_max_ages = numeric(0),
     # misc
+    custom_demography = FALSE,
```
Comment (htopazian) [inline R/parameters.R:410]: I think we should reference the average_age variable here (line #91). If `custom_demography = FALSE` then the model uses average_age = 7663 which is the exponential distribution, right? So does `custom_demography = TRUE` override the average_age argument?

File: vignettes/Demography.Rmd (hunk 1 above)
Comment (giovannic) [reply vignettes/Demography.Rmd:18]: 😄 we had ggplot all along in DESCRIPTION! Not the whole of tidyverse though. I could remove it? 😛

**Pull request merged**